### PR TITLE
ci: migrate to orb tools 11

### PR DESCRIPTION
**Pull request by EricRibeiro**

### CODE REVIEW

File: .circleci/test-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+          device-name: "iPhone 11"
+          device-platform: "iOS"
+          device-version: "13.7"
+          executor:
+            name: "macos"
+            xcode: "11.7.0"
+          filters: *filters
+      - orb-tools/pack:
+          filters: *filters
+      - orb-tools/publish:
```
Comment (KyleTryon) [inline .circleci/test-deploy.yml:103]: For both publishing jobs, add `github-token: GHI_TOKEN`

**Review by KyleTryon — CHANGES_REQUESTED** [review]

**Comment by orb-publisher** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/macos?version=dev:6e460438525cc4c17f9d2aaf8c7ff13208e413b7

**Review by KyleTryon — APPROVED** [review]

**Review by KyleTryon — CHANGES_REQUESTED** [review]
Do not skip rc009

**Comment by EricRibeiro** [comment]
> Do not skip rc009

I have an issue (#21) open to getting it done. The exclude is in there to keep this PR concise.

**Review by KyleTryon — APPROVED** [review]

**Comment by orb-publisher** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/macos?version=dev:9df33f7b9d1011009adf06f87dab6e0311700ac5

**Pull request merged**